### DISCUSSION TITLE
Why are MediaCovers for Sonarr/Radarr/Lidarr defined in the docker compose volumes?

### DISCUSSION THREAD

**Discussion by Briare0s** [htpcBeginner/docker-traefik#147] [Q&A] [answered]
Is there a special reason for this?

**Comment by SimpleHomelab** [comment, marked as answer]
Yes.

I recently moved my home server to proxmox. The whole server now runs in a Ubuntu VM. I wanted to keep the size of the backups small but still be able to restore full functionality in case of a failure.

MediaCovers, Plex metadata, etc. take up several gigabytes of space. Plus they do not compress well. They can be recreated easily if lost.

So I moved those to a separate VM disk that is not backed up. The folder from this VM disk is mounted as a volume. I will clarify this in my docker compose.

The size of my backups went down from 51 GB to 5.6 GB this way. 

For a normal install, this is not needed.

**Reply by Briare0s** [reply]
Thanks for the clarification, that makes a lot sense. Also, thanks a lot for the guides and documentation, it's a great help.
